test/metabase/api/search_test.clj

```diff
@@ -385,8 +391,10 @@
                              (into
                               (default-results-with-collection)
                               (map #(merge default-search-row % (table-search-results))
-                                   [{:name "metric test2 metric", :description "Lookin' for a blueberry", :model "metric"}
-                                    {:name "segment test2 segment", :description "Lookin' for a blueberry", :model "segment"}]))))
+                                   [{:name "metric test2 metric", :description "Lookin' for a blueberry",
+                                     :model "metric" :creator_id true :creator_common_name "Rasta Toucan"}
```

Comment:
if you look into the function `search-request-data`, it passes the data through `mt/boolean-ids-and-timestamps` which basically will turn `creator_id` from int to `boolean` if it exists.

it's way for when we want to check sth exists rather than checking the value explicit